### PR TITLE
List claims command tests

### PR DESCRIPTION
**Pull request by yanqiyqh**
Add JUnit testing for Claim and ListClaimsCommand - ClaimTest and ListClaimsCommandTest

### CODE REVIEW

**Comment by yanqiyqh** [comment]
For #106

File: src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        ListClaimsCommand listClaimsCommand = new ListClaimsCommand(INDEX_FIRST_CLIENT);
+
+        String expectedMessage = String.format(ListClaimsCommand.MESSAGE_SUCCESS,
+                clientToDisplay.getName().toString(),
+                clientToDisplay.getInsurancePlansManager().accessClaims());
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        assertCommandSuccess(listClaimsCommand, model, expectedMessage, expectedModel);
+    }
```
Comment (RezwanAhmed123) [inline src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java:42]: Maybe some javadoc could be written here and for `validIndexFilteredList` as it's not clear what these tests are testing right now?

**Review by RezwanAhmed123 — COMMENTED** [review]
LGTM, some nits.

File: src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java (hunk 1 above)
Comment (yanqiyqh) [reply src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java:42]: I agree, I will add javadoc to make it clearer.

Comment (RezwanAhmed123) [reply src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java:60]: alright understood!

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM

**Pull request merged**